apps/studio/components/layouts/ProjectLayout/ProjectLayout.tsx

```diff
@@ -233,7 +213,10 @@ const ProjectLayout = forwardRef<HTMLDivElement, PropsWithChildren<ProjectLayout
                     </ResizablePanel>
                     {isClient && aiAssistantPanel.open && (
                       <>
-                        <ResizableHandle />
+                        <ResizableHandle
+                          withHandle
+                          className="opacity-0 focus-within:opacity-100 hover:opacity-100"
+                        />
                         <ResizablePanel
```

Comment:
The AI Assistant is not really resizable (you can't expand it) so this could be just a regular div.

Reply:
I'll remove the handle and leave the rest ready for a follow up